test/amp_tags_test.js

```diff
@@ -74,6 +112,18 @@ describe('AmpTags', function () {
     assertContains('<link rel="amphtml" href="https://madrid.quintype.io/amp/story/section%2Fslug"/>', string);
   });
 
+  it("does append domain to amp visual stories if appendHostToAmpUrl present", function () {
+    const story = { slug: "section/slug", "is-amp-supported": true };
+    const string = getSeoMetadata(
+      { ...seoConfig, appendHostToAmpUrl: true },
+      config,
+      "story-page",
+      { currentHostUrl: "https://madrid.quintype.io/section/slug", data: { story: { ...story, "story-template": "visual-story" } } },
+      {}
+    );
+    assertContains('<link rel="amphtml" href="https://madrid.quintype.io/section%2Fslug"/>', string);
+  });
+
```

Comment:
independent of story template, test is not needed

Reply:
removed